internal/ingress/controller/controller_test.go

```diff
@@ -2468,6 +2468,149 @@ func TestGetBackendServers(t *testing.T) {
 				}
 			},
 		},
+		{
+			Ingresses: []*ingress.Ingress{
+				{
+					Ingress: networking.Ingress{
+						ObjectMeta: metav1.ObjectMeta{
+							Namespace: "example",
+						},
+						Spec: networking.IngressSpec{
+							Rules: []networking.IngressRule{
+								{
+									Host: "example.com",
```

Comment:
Should I make a `const` for this as the linter suggests? It doesn't seem to match the way the existing tests are written.

Reply:
yes please, let's make linter happy